src/pages/Account/Login.tsx

```diff
@@ -1,17 +1,28 @@
+import React, { useState } from "react";
+
+import { SSO_LOGIN_ICON } from "settings";
+
+import { useSSO } from "hooks/useSSO";
+
 import LoginForm from "components/organisms/AuthenticationModal/LoginForm";
 import PasswordResetForm from "components/organisms/AuthenticationModal/PasswordResetForm";
 import RegisterForm from "components/organisms/AuthenticationModal/RegisterForm";
 import { InitialForm } from "components/organisms/AuthenticationModal/InitialForm";
-import React, { FC, useState } from "react";
+
 import "./Account.scss";
+import "./Login.scss";
```

Comment:
If that `Account.scss` file isn't being used anywhere else (which ideally it shouldn't be) then we should renaming it to `Login.scss` and combine it with the new styles introduced here.

At an absolute minimum, if we can't do that, we should add an `@debt` comment to do so in future.

Reply:
![image](https://user-images.githubusercontent.com/63194656/122247279-e88c6800-cecf-11eb-96d1-90123865d9ae.png)

Reply:
Comment added in https://github.com/sparkletown/sparkle/pull/1546/commits/4fe8c301fc88ac30c81c63af80e6ab29fcbe538b